ninja-servlet-integration-test/Dockerfile.jetty

```diff
@@ -0,0 +1,4 @@
+# 9.3.15, 9.4
```

Comment:
(not sure if this comes later) but it may be beneficial having a README saying exactly for what these Dockerfiles can be used. I know it now... But I know myself as well an in 1 year from now I'll have completely forgotten about them and will wonder why these exist...

Reply:
I debated with myself on not even including these on the PR.  However, it was pretty useful to test our .war against a few servlet impls. For example, I discovered Ninja's cookie handling using an equals char isn't supported by Tomcat & Wildfly.  I'll add a README to explain why we have them.  IMHO, I think it may be useful to rework this module to allow for testing it against a number of impls down the road.